server/Assignments_filler/sheetFiller.py

```diff
@@ -0,0 +1,237 @@
+    if amount == 1:
+        values = [[data]]
+    if amount == 3:
+        values = [[data[0], data[1], data[2]]]
+    body = {
+        'values': values
+    }
+    result = service.spreadsheets().values().update(
+        spreadsheetId=SPREADSHEET_ID, range=range,
+        valueInputOption='USER_ENTERED', body=body).execute()
+
+def checker(service, contractName, nid, numberOfComments):
+    readRange = TEAM_ASSINGMENT_TAB+RANGE_READ
+    oldContractName = contractName
+    oldNID = nid
+    query = ("SELECT `created` FROM drupal.comment where nid=%s ORDER BY created DESC" % (oldNID))
+    oldCommentList = get_from_db(query)
+    oldLastUpdated = oldCommentList[0][0]
+    oldNumberOfComments = numberOfComments
+    
+    logging.info("Starting Checker")
+
+    while True:
+        #read new contract name from DB
+        writtenName = readSheet(service, readRange)
+        query = ("SELECT `nid`,`title` FROM drupal.node where title like \"%%%s%%\"" % (writtenName))
+        contractList = get_from_db(query)
+        newNID = contractList[0][0]
+        newContractName = contractList[0][1]
+
+        #check last updated comment and number of comments
+        query = ("SELECT `created` FROM drupal.comment where nid=%s ORDER BY created DESC" % (newNID))
+        commentList = get_from_db(query)
+        newLastUpdated = commentList[0][0]
+        newNumberOfComments = len(commentList)
+        
+        #compare the new and the old data
+        if (newContractName != oldContractName):
+            logging.info("New contract name detected!")
+            nameChanged = True
+            break
+        elif (newNumberOfComments != oldNumberOfComments):
+            logging.info("A comment was added!")
+            break
+        elif (newLastUpdated != oldLastUpdated):
+            logging.info("A comment was modified!")
+            break
+        #print ("looping")
+        time.sleep(refreshRate)
+    
+    filler(service, nameChanged, newContractName, newNID)
+
+def filler(service, nameChanged, contractName, nid):
+    logging.info("Running Filler")
+
+    readRange = TEAM_ASSINGMENT_TAB+RANGE_READ
+
+    #fill mission full name
+    if nameChanged:
+        writeSheet(service, contractName, readRange, 1)
+        logging.info("Filled Mission Name : %s" % (contractName))
+    
+    query = ("SELECT `cid`,`uid`,`subject` FROM drupal.comment where nid=%s ORDER BY cid ASC" % (nid))
+    commentList = get_from_db(query)
+    numberOfComments = len(commentList)
+    row = 2
+    
+    #save a list of the current comments
+    nameRange = TEAM_ASSINGMENT_TAB + "B2:B" + str(numberOfComments+1)
+    badListOfNames = readSheet(service, nameRange, 2)
+    goodListOfNames = []
+    for sublist in badListOfNames:
+        goodListOfNames.extend(sublist)
+    commentsRange = TEAM_ASSINGMENT_TAB + "D2:D" + str(numberOfComments+1)
+    listOfComments = readSheet(service, commentsRange, 2)
+    print (goodListOfNames)
```

Comment:
```suggestion
    print(goodListOfNames)
```
No space between function name and parentheses.